pkgs/applications/misc/get_iplayer/default.nix

```diff
@@ -33,10 +33,9 @@ perlPackages.buildPerlPackage rec {
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/bin $out/share/man/man1
-    cp get_iplayer $out/bin
+    install -D get_iplayer -t $out/bin
     wrapProgram $out/bin/get_iplayer --suffix PATH : ${lib.makeBinPath [ atomicparsley ffmpeg ]} --prefix PERL5LIB : $PERL5LIB
-    cp get_iplayer.1 $out/share/man/man1
+    install -D get_iplayer.1 -t $out/share/man/man1
```

Comment:
or use `installManPage` from `installShellFiles`

Reply:
Didn't know about that option. I think I'll stick with plain `install` though to avoid having to add in another dependency. Many thanks!

Reply:
Just some clever shell wrapper, I'm happy with however you see fit.